capstone-frontend/src/components/UserCard.jsx

```diff
@@ -7,36 +7,78 @@ import '../styles/Groups.css';
 class UserCard extends Component {
   removeMember = () => {
     const jsonBody = {
-      "remove_memberIDs": this.props.user.id, 
+      "remove_memberIDs": this.props.user.id,
       "id": this.props.club.id
     }
-    fetch(`/api/clubs`, {
-        method: "put", 
-        body: JSON.stringify(jsonBody)
-      })
+    fetch(`/api/clubs`, { method: "put", body: JSON.stringify(jsonBody) })
```

Comment:
nit: If you are not using a variable inside the link to the API, you can use single quotes instead of the backticks